GitUI/CommandsDialogs/FormRebase.cs

```diff
@@ -381,17 +381,22 @@ private void btnChooseFromRevision_Click(object sender, EventArgs e)
             bool previousValueBranchFilterEnabled = AppSettings.BranchFilterEnabled;
             bool previousValueShowCurrentBranchOnly = AppSettings.ShowCurrentBranchOnly;
             bool previousValueShowReflogReferences = AppSettings.ShowReflogReferences;
+            bool previousValueShowStashes = AppSettings.ShowStashes;
 
             try
             {
-                using FormChooseCommit chooseForm = new(UICommands, txtFrom.Text, showCurrentBranchOnly: true);
+                AppSettings.ShowStashes = false;
+                ObjectId firstParent = UICommands.GitModule.RevParse("HEAD~");
+                string preSelectedCommit = string.IsNullOrEmpty(txtFrom.Text) ? firstParent?.ToString() ?? string.Empty : txtFrom.Text;
```

Comment:
`Text` value can't be null, but it can whitespace. So the check should be `IsNullOrWhitespace`. 
Also suggest to invert the condition as it'd read easier.